app/testOnly/views/helpers/inputHelper.scala.html

```diff
@@ -47,7 +47,7 @@
         @formHint.fold(Nil: Seq[Html])(_.map(hintText))
     </label>
     @fieldErrorHelper(field, parentForm)
-    <input name=@field.name class="input--fullwidth" id=@field.name type="text"
+    <input name=@field.name class="input--fullwidth" id=@field.name type="text" aria-describedby="input-helper-hint"
```

Comment:
hint text is optional, if we have an input without hint text, this will have aria-describedby linking to something which doesn't exist.

Reply:
Done